src/smirnoffio.py

```diff
@@ -283,7 +272,7 @@ def prepare(self, pbc=False, mmopts={}, **kwargs):
         openff_mols = []
         for fnm in self.mol2:
             try:
-                mol = OffMolecule.from_file(fnm, allow_undefined_stereo=True)
+                mol = OffMolecule.from_file(fnm)
             except Exception as e:
```

Comment:
See earlier comment for more specific `UndefinedStereochemistryError` class path